### PR TITLE
Fix icons on activity list

### PR DESCRIPTION
**Pull request by grilix**
- Show collectible icons
- Show asset icons when address is not on contacts

## Activity list

activity-images

### CODE REVIEW

File: src/actions/collectiblesActions.js [hunk 1]
```diff
@@ -46,19 +46,22 @@ export const fetchCollectiblesAction = () => {
...
         description,
-        image_preview_url: image,
+        image_url: fullImage,
+        image_preview_url: preview,
         image_thumbnail_url: thumbnail,
       } = collectible;
       const { name: category, address: contractAddress } = assetContract;
       const collectibleName = name || `${category} ${id}`;
 
+      const image = (/\.(png|gif)$/i).test(fullImage) ? fullImage : '';
```
Comment (jsidorenko) [inline src/actions/collectiblesActions.js]: Shall we test `preview` for extensions as well? The problem here is that some images are stored on ipfs and we have no way to display them

File: src/components/ActivityFeed/ActivityFeed.js [hunk 2]
```diff
@@ -223,16 +227,21 @@ class ActivityFeed extends React.Component<Props, State> {
         directionSymbol = '';
       }
 
+      const fullIconUrl = iconUrl ? `${SDK_PROVIDER}/${iconUrl}?size=3` : '';
+
       const contact = contacts
         .find(({ ethAddress }) => address.toUpperCase() === ethAddress.toUpperCase()) || {};
+      const inContacts = Object.keys(contact).length !== 0;
```
Comment (jsidorenko) [inline src/components/ActivityFeed/ActivityFeed.js]:
```suggestion
      const isContact = Object.keys(contact).length !== 0;
```

File: src/components/ActivityFeed/ActivityFeed.js [hunk 3]
```diff
@@ -223,16 +227,21 @@ class ActivityFeed extends React.Component<Props, State> {
...
         <ListItemWithImage
           onPress={() => this.selectEvent({ ...notification, value, contact }, type, notification.status)}
           label={nameOrAddress}
-          avatarUrl={contact.profileImage}
-          navigateToProfile={Object.keys(contact).length !== 0 ? navigateToContact : null}
-          iconName={Object.keys(contact).length === 0 || showArrowsOnly ? directionIcon : null}
+          avatarUrl={itemImage}
+          navigateToProfile={inContacts ? navigateToContact : null}
+          iconName={(showArrowsOnly || !itemImage) ? directionIcon : null}
+          imageAddonIconName={(itemImage && !showArrowsOnly) ? directionIcon : undefined}
```
Comment (jsidorenko) [inline src/components/ActivityFeed/ActivityFeed.js]: Shouldn't it be `null` and not `undefined`?

File: src/actions/collectiblesActions.js (hunk 1 above)
Comment (grilix) [reply src/actions/collectiblesActions.js]: Oh, okay, we can test that as well. Will update

File: src/components/ActivityFeed/ActivityFeed.js (hunk 3 above)
Comment (grilix) [reply src/components/ActivityFeed/ActivityFeed.js]: With `null` it triggers a warning for incompatible types. I just copied how we do it on other places. We could update the `ListItemWithImage` component to allow `imageAddonIconName=null`, but  I think it's not on the scope of this fix. What do you think?

Comment (jsidorenko) [reply src/components/ActivityFeed/ActivityFeed.js]: No, it's fine with undefined :)

File: src/components/ActivityFeed/ActivityFeed.js (hunk 2 above)
Comment (grilix) [reply src/components/ActivityFeed/ActivityFeed.js]: Updated

File: src/actions/collectiblesActions.js (hunk 1 above)
Comment (grilix) [reply src/actions/collectiblesActions.js]: Updated

**Pull request merged**